<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QComboBox" name="languageKeyComboBox">
     <item>
      <property name="text">
       <string>Hieroglyphics</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unknown Pokemon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Morse Code</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Wingdings</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PigPen</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QScrollArea" name="keyScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>624</width>
        <height>800</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="languageKeyLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/translation_images/hieroglyphics.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QScrollArea" name="selectedImageScrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>358</width>
          <height>230</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="selectedImageLabel">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Resource/puppy.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="selectImageButton">
     <property name="text">
      <string>Select Image</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="translateMorseCodeButton">
     <property name="text">
      <string>Translate Morse Code</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="translation_images.qrc"/>
 </resources>
 <connections/>
</ui>
